tests/unit/test_vedabase.py

```diff
@@ -4,17 +4,97 @@
 os.environ["SANDMAN_API"] = "https://veda-api-development.geobigdata.io"
 
 from pyveda import VedaCollection
-from pyveda.db import VedaBase
+from pyveda.vedaset import VedaBase
+from pyveda.fetch.handlers import ClassificationHandler, SegmentationHandler, ObjDetectionHandler
 import json
 from shapely.geometry import shape, box
 from shapely.geometry.polygon import Polygon
+from auth_mock import gbdx
+import vcr
 import unittest
+from unittest import skip
 
+def force(r1, r2):
+    return True
+
+my_vcr = vcr.VCR()
+my_vcr.register_matcher('force', force)
+my_vcr.match_on = ['force']
+
+class VedaBaseLabelTest(unittest.TestCase):
+    def test_classification(self):
+        item = {
```

Comment:
useful having these "test" responses for the various datas. Could we move them into a test-data module or something and import them from there instead on defining in the test?